climakitae/explore/agnostic.py

```diff
@@ -362,10 +383,8 @@ def _compute_results(selections, metric, years, months):
 
     # Retrieving closest grid-cell's data for lat/lon area subsetting
     if selections.area_subset == "lat/lon":

```

Comment:
Users of `agg_area_subset_sims` could potentially subset area by “lat/lon”, in which case I think `agg_area_subset_sims `intends to do computation on all data?

Reply:
Do you mean in the case where all the user wants to select on a bounding box of lat/lons?

Reply:
Yes, for such users, if any, is the calculation intended to incorporate all grid cells in the bounding box, instead of the one grid cell in the middle?

Reply:
This is a good point, I think this may be a question for development standup. Currently, I believe the scope of the function is to still just handle individual gridpoints, but as users will want to include assets that span multiple gridcells, I can imagine that we would need to modify the function to handle these cases too.